docs/classic-ui/page-templates/index.md

```diff
@@ -0,0 +1,196 @@
+# Page Templates in Plone Classic-UI
+
+These instructions and resources will help you develop Classic-UI pages.
+
+When you visit the Plone backend with a web browser, you will reach the Classic-UI.  These pages are generated with Classic Plone Browser Views and Page Templates.
```

Comment:
I don't think all these words should be capitalized, unless they are proper nouns:

"Classic Plone Browser Views and Page Templates"

...and throughout this page.

Reply:
Maybe not classic, but the other terms like Browser View or Page Templates are Zope specific terms, but you knoe better.
Please make a suggestion. We should get these chapter in as soon as possible.

Reply:
Use this quick definition for a [proper noun](https://duckduckgo.com/?q=what+is+a+proper+noun&atb=v314-1&ia=definition), and it might become clearer. "Zope" and "Plone" are proper nouns. Many web frameworks use the terms "browser views" and "page templates", and are not particular. Even [Zope's own documentation](https://www.zope.dev/documentation/zope_secrets/request.html?highlight=page%20templates) lower cases these terms. Only if you are certain that something is a proper noun and you would swear on your beloved nana's grave that it is so, then it should be title-cased, else use lower case.